mage_ai/frontend/components/Triggers/utils.ts

```diff
@@ -205,11 +205,7 @@ export function getTriggerApiEndpoint(pipelineSchedule: PipelineScheduleType) {
 
   const windowIsDefined = typeof window !== 'undefined';
   if (windowIsDefined) {
-    url = `${window.origin}/api/pipeline_schedules/${pipelineSchedule?.id}/pipeline_runs`;
```

Comment:
Can we provider both ways for API trigger?
Some webhooks can't be easily configured to use headers

Reply:
sure i can do that